<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-02175GH" failures="0" tests="5" name="com.SwagLabs.TestCases.InventoryTest" time="0.001" errors="1" timestamp="2024-03-11T04:51:18 PDT" skipped="4">
  <testcase classname="com.SwagLabs.TestCases.InventoryTest" name="beforeclasssetup" time="0.001">
    <error message="Cannot invoke &quot;com.SwagLabs.Pages.LoginPage.getapptitle()&quot; because &quot;this.login&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.SwagLabs.Pages.LoginPage.getapptitle()" because "this.login" is null
at com.SwagLabs.TestCases.BaseClass.beforeclasssetup(BaseClass.java:44)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:180)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- beforeclasssetup -->
  <system-out/>
  <testcase classname="com.SwagLabs.TestCases.InventoryTest" name="verifypcount" time="0.000">
    <skipped/>
  </testcase> <!-- verifypcount -->
  <system-out/>
  <testcase classname="com.SwagLabs.TestCases.InventoryTest" name="verifylable" time="0.000">
    <skipped/>
  </testcase> <!-- verifylable -->
  <system-out/>
  <testcase classname="com.SwagLabs.TestCases.InventoryTest" name="selectproduct" time="0.000">
    <skipped/>
  </testcase> <!-- selectproduct -->
  <system-out/>
  <testcase classname="com.SwagLabs.TestCases.InventoryTest" name="verifyproduct" time="0.000">
    <skipped/>
  </testcase> <!-- verifyproduct -->
  <system-out/>
</testsuite> <!-- com.SwagLabs.TestCases.InventoryTest -->
